src/nvim/api/win_config.c

```diff
@@ -10,16 +10,19 @@
 #include "nvim/api/private/helpers.h"
 #include "nvim/api/win_config.h"
 #include "nvim/ascii.h"
+#include "nvim/buffer_defs.h"
 #include "nvim/drawscreen.h"
 #include "nvim/highlight_group.h"
 #include "nvim/option.h"
 #include "nvim/strings.h"
 #include "nvim/syntax.h"
 #include "nvim/ui.h"
 #include "nvim/window.h"
+#include <inttypes.h>
 
 #ifdef INCLUDE_GENERATED_DECLARATIONS
 # include "api/win_config.c.generated.h"
+# include "api/extmark.h.generated.h"
```

Comment:
It's included in `extmark.h`. Move it out of this `ifdef`
```suggestion
#include "nvim/api/extmark.h"
```